packages/cli/package-lock.json

```diff
@@ -0,0 +1,4683 @@
+{
```

Comment:
I presume that these package-lock changes are done by `lerna bootstrap`? Ideally we'd be able to verify that our dependency versions are still the same, but these diffs seem a little too large to manually verify 🙁

Reply:
yep, I think that if our functional tests stay the same we should be good though, since `package-lock` is only effective in development environment

if there are changes (like the font awesome bundles), we can then investigate them and deal with the diff in a separate pr